packages/admin-next/dashboard/src/components/layout-v2/main-layout/main-layout.tsx

```diff
@@ -117,8 +117,16 @@ const useCoreRoutes = (): Omit<NavItemProps, "pathname">[] => {
       to: "/inventory",
       items: [
         {
-          label: t("reservations.domain"),
-          to: "/reservations",
+          label: t("locations.domain"),
+          to: "/inventory/locations",
+        },
+        {
+          label: t("transfers.domain"),
+          to: "/inventory/transfers",
+        },
+        {
+          label: t("suppliers.domain"),
+          to: "/inventory/suppliers",
```

Comment:
q: what is "Suppliers"? Can't find any reference to it in any of the modules. Of course depending on what it is, but intuitively it seems like a settings page to me.

Reply:
this was in the design, but it's a placeholder for now anyways, the point of this PR was to allow for working on each tab in parallel potentially